src/main/java/mate/academy/service/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.service;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/library";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "Gfhfyjhvf2003_1995");
```

Comment:
Storing database credentials directly in the code is a security risk. Consider using environment variables or a secure configuration file to store sensitive information like database usernames and passwords.